src/App.tsx

```diff
@@ -1,14 +1,50 @@
 /* eslint-disable max-len */
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 
+import { initialTodo, initialUser } from './initialState/initialState';
+import { getTodos, getUser } from './api';
+import { getSortedTodos } from './helpers/getSortedTodos';
 import { TodoList } from './components/TodoList';
 import { TodoFilter } from './components/TodoFilter';
 import { TodoModal } from './components/TodoModal';
 import { Loader } from './components/Loader';
+import { Todo } from './types/Todo';
 
 export const App: React.FC = () => {
+  const [todoList, setTodoList] = useState<Todo[]>([]);
+  const [loading, setLoading] = useState(false);
+  const [loadingModal, setLoadingModal] = useState(false);
+  const [modalShow, setModalShow] = useState(false);
+  const [activeTodo, setActiveTodo] = useState(initialTodo);
+  const [userTodo, setUserTodo] = useState(initialUser);
```

Comment:
Better choice for name would be `todoUser`, `todoUsername`, `todoOwner` or `todoAssignee`, depending on the meaning of user entity in the app. `userTodo` sound more like user's TODO.
Also it makes sense to indicate `User` type here.

Reply:
Thank you)